node/src/chain_spec.rs

```diff
@@ -146,13 +147,21 @@ pub fn development_config(
     para_id: ParaId,
     container_chains: Vec<String>,
     mock_container_chains: Vec<ParaId>,
+    boot_nodes: Vec<String>,
 ) -> ChainSpec {
     // Give your base currency a unit name and decimal places
     let mut properties = sc_chain_spec::Properties::new();
     properties.insert("tokenSymbol".into(), "UNIT".into());
     properties.insert("tokenDecimals".into(), 12.into());
     properties.insert("ss58Format".into(), 42.into());
     properties.insert("isEthereum".into(), false.into());
+    let boot_nodes: Vec<MultiaddrWithPeerId> = boot_nodes
```

Comment:
Why do we need this in the orchestrator?  I understand that for containers you need to generate a chain-spec with specific bootnodes on it to be able to register it properly in the orchestrator, but I dont see a reason why this would be needed in the orchestrator

Reply:
I added it for "symmetry", we can remove it.

Reply:
Yeah I think it does not make much sense for now  in Tanssi.